README.md

```diff
@@ -603,6 +603,24 @@ log = starkbank.invoice.log.get("5155165527080960")
 print(log)
 ```
 
+### Get a reversed invoice log PDF
+
+After its reversal, a reversed invoice log PDF may be retrieved by its id. 
```

Comment:
Acho que ficou um pouco confuso, colocaria algo como:
"After any successful reversal applied to an Invoice, a reversed log will be created. To retrieve a specific reversal receipt, you can request the corresponding log PDF:"